web/yo/app/scripts/controllers/tools.js

```diff
@@ -250,13 +270,20 @@ angular.module('oncokbApp')
             }
             $scope.generateEvidences = function () {
                 $scope.loadingReviewed = true;
-                $scope.reviewedData.geneType.body = [];
-                $scope.reviewedData.mutationEffect.body = [];
-                $scope.reviewedData.tumorSummary.body = [];
-                $scope.reviewedData.drugs.body = [];
 
                 DatabaseConnector.getReviewedData($scope.reviewedData[$scope.evidenceType].evidenceTypes).then(function(response) {
-                    if ($scope.evidenceType === 'geneType') {
+                    if ($scope.evidenceType === 'geneSummary' || $scope.evidenceType === 'geneBackground') {
+                        // key = 'summary' or key = 'background'
```

Comment:
No longer needed?

Reply:
No, we still need `geneType` and it's on line 286. I also sent screenshots to Sarah and she thought that's good to release.